Makefile

```diff
@@ -599,25 +1024,29 @@ endif
 
 .PHONY: makeruntime
 makeruntime:
-	$(MAKE) -C runtime $(BOOT_FLEXLINK_CMD) all
-runtime/libcamlrun.$(A): makeruntime ;
+	$(MAKE) $(BOOT_FLEXLINK_CMD) runtime-all
 stdlib/libcamlrun.$(A): runtime/libcamlrun.$(A)
 	cd stdlib; $(LN) ../runtime/libcamlrun.$(A) .
 clean::
-	$(MAKE) -C runtime clean
+	rm -f $(addprefix runtime/, *.o *.obj *.a *.lib *.so *.dll ld.conf)
+	rm -f $(addprefix runtime/, ocamlrun ocamlrund ocamlruni ocamlruns sak)
+	rm -f $(addprefix runtime/, \
+	  ocamlrun.exe ocamlrund.exe ocamlruni.exe ocamlruns.exe sak.exe)
+	rm -f runtime/primitives runtime/primitives.new runtime/prims.c \
+	  $(runtime_BUILT_HEADERS)
+	rm -f runtime/domain_state*.inc
+	rm -rf $(DEPDIR)
```

Comment:
I think it's fine, but just noting that this erases all dependencies, not just `$(DEPDIR)/runtime`